### DISCUSSION TITLE
Manifest not found & Manifest unknown when creating container for etcd using Docker

### DISCUSSION THREAD

**Discussion by dev-vaayen** [etcd-io/etcd#16118] [Q&A] [answered]
Hi everyone

I am using the https://etcd.io/docs/v3.5/op-guide/container/ document for creating clusters in a container. In the second step for creating a container using Docker, the following command has been provided:
```
REGISTRY=quay.io/coreos/etcd
# available from v3.2.5
REGISTRY=gcr.io/etcd-development/etcd

docker run \
  -p 2379:2379 \
  -p 2380:2380 \
  --volume=${DATA_DIR}:/etcd-data \
  --name etcd ${REGISTRY}:latest \
  /usr/local/bin/etcd \
  --data-dir=/etcd-data --name node1 \
  --initial-advertise-peer-urls http://${NODE1}:2380 --listen-peer-urls http://0.0.0.0:2380 \
  --advertise-client-urls http://${NODE1}:2379 --listen-client-urls http://0.0.0.0:2379 \
  --initial-cluster node1=http://${NODE1}:2380
```

The issue is that, when I try to run this command in my terminal, I get the following output:
```
Unable to find image 'gcr.io/etcd-development/etcd:latest' locally
docker: Error response from daemon: manifest for gcr.io/etcd-development/etcd:latest not found: manifest unknown: Failed to fetch "latest" from request "/v2/etcd-development/etcd/manifests/latest".
See 'docker run --help'.
```
Please let me know why this is occurring and how I create the container successfully.

Thank You.

**Comment by jmhbnz** [comment, marked as answer]
Hey @Anand11Yash thanks for your question, unfortunately we have some outdated documentation here, refer this issue: https://github.com/etcd-io/website/issues/271 

Try image tag `v3.5.9` instead of `latest`.

Please feel free to raise a documentation fix pull request if you have a moment, we need some help to get that page fixed! 🙏🏻

**Reply by dev-vaayen** [reply]
Hi @jmhbnz - thank you so much for such a quick response! I followed your suggestion and the container has been created now. 

For reference for anyone else who might face the same problem as me, the modified command that got my container running was as follows:
```
REGISTRY=quay.io/coreos/etcd:v3.5.9

docker run \
-p 2379:2379 \
-p 2380:2380 \
--volume=${DATA_DIR}:/etcd-data \
--name etcd ${REGISTRY} \
/usr/local/bin/etcd \
--data-dir=/etcd-data --name node1 \
--initial-advertise-peer-urls http://my.ip.add.ress:2380 --listen-peer-urls http://0.0.0.0:2380 \
--advertise-client-urls http://my.ip.add.ress:2379 --listen-client-urls http://0.0.0.0:2379 \
--initial-cluster node1=http://my.ip.add.ress:2380
```
Please note that If I had only added v3.5.9 to the code provided in the documentation, it would have only created a container for the image of that version. However, to connect the container I had to further modify that code by adding my IP address to get the first etcd node running through ```etcdctl --endpoints=http://${NODE1}:2379 member list"```.

Regarding fixing the documentation, I would love to contribute in anyway that I can! I will be sure to make a documentation fix pull request soon.
